handler.cpp

```diff
@@ -441,3 +442,75 @@ RequestHandler::Status ProxyHandler::HandleRequest(const Request& request, Respo
     }
     return RequestHandler::PASS;
 }
+
+
+//////////////////////////////////////////////////////////////////////////////////////
+/////////////////////////////////// Location Handler /////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////////////
+
+RequestHandler::Status LocationHandler::Init(const std::string& uri_prefix, const NginxConfig& config) {
+    return RequestHandler::PASS;
+}
+
+RequestHandler::Status LocationHandler::HandleRequest(const Request& request, Response* response) {
+    std::ifstream infile("location_log.txt");
+
+    if (!infile.is_open())
+        return RequestHandler::FAIL;
+
+    location_frequency.clear();
```

Comment:
yeah it's a little inefficient, but our thing isnt build to handle like crazy scale anyway, so I wrote it this way to make it thread safe, etc.